sycl/doc/extensions/experimental/sycl_ext_oneapi_graph.asciidoc

```diff
@@ -751,6 +751,25 @@ dynamic parameter for the buffer can be registered with all the nodes which
 use the buffer as a parameter. Then a single `dynamic_parameter::update()` call
 will maintain the graphs data dependencies.
 
+===== Whole Graph Update
+
+A graph in the executable state can have all of its nodes inputs & outputs updated
+using the `command_graph<graph_state::executable>::update()` method. This method takes
```

Comment:
we've got a few overloads of `update()` so might be better to be more verbose and put in the graph parameter. Likewise on like 736, might want to update to `update(nodes)`

Reply:
Done :+1: . It feels a bit too verbose now (one function takes the whole line). But I agree that it's better to be clear about what we are talking about.

Reply:
know what you mean, we could just do `command_graph<graph_state::executable>::update(graph)` here  rather than the full    `command_graph<graph_state::executable>::update(const command_graph<graph_state::modifiable>& graph)` and that would be enough to disambiguate from the node update.